google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,17 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    // Present a canonical target folder as default in the path field.
+    if (project != null && project.getBasePath() != null) {
+      if (fileType == ConfigFileType.APP_YAML) {
+        destinationFolderChooser.setText(project.getBasePath() + "/src/main/appengine");
```

Comment:
this is kind of fragile. multi-module projects and projects that don't use the maven directory structure won't like this..This is actually a tricky problem because for the artifact types we support we don't necessarily know what module it's related to so we can't query for the src paths.

I still think it's better than no suggestion, and it gives the user an indication of where they should be. 

:ship:  LGTM